### DISCUSSION TITLE
Install zookeeper without kafka

### DISCUSSION THREAD

**Discussion by eugene-chernyshenko** [strimzi/strimzi-kafka-operator#8591] [Q&A]
Is it possible to install zookeeper without kafka? 
For example clickhouse needs zookeeper for replication

**Comment by scholzj** [comment]
No, Strimzi does not support ZooKeeper deployment without Kafka. ZooKeeper is basically an implementation detail for us and it will be removed once Kafka does not need it (likely some time next year).
